### PR TITLE
Fix sysmod integration tests on Windows

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?

Fixes sysmod integration tests on Windows. Adds missing CLI Examples and Docstrings... which is apparently part of the sysmod test.

https://github.com/saltstack/zh/issues/853

### CODE REVIEW

File: salt/modules/win_dacl.py [hunk 1]
```diff
@@ -341,7 +341,20 @@ def _get_dacl(path, objectType):
 
 def get(path, objectType, user=None):
     '''
-    Get the acl of an object. Will filter by user if one is provided.
+    Get the acl ef an object. Will filter by user if one is provided.
```
Comment (cachedout) [inline salt/modules/win_dacl.py]: ACL not acl.

File: salt/modules/win_system.py [hunk 2]
```diff
@@ -300,7 +300,13 @@ def shutdown_abort():
     displayed to the user. Once the shutdown has initiated, it cannot be aborted
 
     :return: True if successful
-    :rtype: bool
+    :rtype: boo
```
Comment (cachedout) [inline salt/modules/win_system.py]: bool

File: salt/modules/win_task.py [hunk 3]
```diff
@@ -800,7 +842,13 @@ def edit_task(name=None,
     - Stop Existing
 
     :return: True if successful, False if unsuccessful
-    :rtype: bool
+    :rtype: boo
```
Comment (cachedout) [inline salt/modules/win_task.py]: bool

**Comment by cachedout** [comment]
@twangboy This will not merge cleanly. Could you please rebase it? Thanks.

**Comment by twangboy** [comment]
rebased

**Pull request merged**